src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/BedAssignmentHistory.java

```diff
@@ -51,7 +69,8 @@ public static class BedAssignmentHistoryPK implements Serializable {
     @Column(name = "LIVING_UNIT_ID")
     private Long livingUnitId;
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @NotFound(action = IGNORE)
```

Comment:
LIVING_UNIT_ID has no foreign key and not all history records have a matching cell due to bad NOMIS data, so this needs to be ignored (which therefore means it can not be LAZY loaded)